### PR TITLE
Update Manifest to not use git clones added in #103

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**
Should fix CI broken in #103

### CODE REVIEW

**Comment by Vaibhavdixit02** [comment]
Looks like it also needs a new token for accessing PumasRegistry @andreasnoack

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,36 +1,50 @@
...
+      # else
+        julia --depwarn=error --project -e 'using Pkg;
+          Pkg.test("PumasTutorials");'
+      # fi
+  interruptible: true
   artifacts:
     paths:
-      - html
-    expire_in: 1 week
+      - Manifest.toml
```
Comment (andreasnoack) [inline .gitlab-ci.yml:36]: What is the motivation for this change?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -1,36 +1,50 @@
...
   # GitLab requires that the cache path is insde the project
   JULIA_DEPOT_PATH: "$CI_PROJECT_DIR/.julia/"
 
 cache:
   paths:
     - .julia/compiled/
 
-test:
+
+.template: &template
```
Comment (andreasnoack) [inline .gitlab-ci.yml:16]: This seems unnecessary when the template is only used once

**Review by andreasnoack — COMMENTED** [review]
Is it useful to store the htmls as artifacts? I thought so yesterday but I'm not sure anymore. If not then maybe we should just drop `test` job.
